### ISSUE TITLE
Add https://github.com/orangeduck/Cello

### ISSUE THREAD

**Issue by stianhoiland** [Hirrolot/awesome-c-preprocessor#4]

**Comment by Hirrolot** [comment]
I was thinking of adding Cello, but wasn't able to find any non-trivial macros within it. Am I missing something?

**Comment by stianhoiland** [comment]
No, you're right. Cello is not a macro lib. I just thought Cello would fit rather well under the "C language extensions" header anyway.

**Comment by Hirrolot** [comment]
Resolved by https://github.com/Hirrolot/awesome-c-preprocessor/commit/e3a8b69abb7ea44424c1112657608a00befe9fc5.
